.github/workflows/publish-release.yml

```
@@ -11,7 +11,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
```

I'm not positive about this project, but I've run into others where Python's setuptools_scm requires regular Git history, not just a shallow checkout of the HEAD commit.

Additionally I've run into some that need the tags present, in which case you also need

```suggestion
          fetch-depth: 0
  - run: |
      git fetch --prune --tags ||:
```